### PR TITLE
New package: SweetHome3D-6.1.2

### PR DESCRIPTION
**Pull request by 1is7ac3**

### CODE REVIEW

File: srcpkgs/SweetHome3D/files/sweethome3d [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    JAVA_EXEC="${JAVA_HOME}/bin/java"
+  fi
+else
+  JAVA_EXEC="$(which java)"
+fi
+
+if [ -z "${JAVA_EXEC}" ]
+then
+  echo "No 'java' executable can be found, please set JAVA_HOME variable or"
+  echo "use the 'archlinux-java' script to set the Java version."
```
Comment (Cogitri) [inline srcpkgs/SweetHome3D/files/sweethome3d]: This doesn't seem right to me

File: srcpkgs/SweetHome3D/template [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Template file for 'SweetHome3D'
+pkgname=SweetHome3D
+version=6.1.2
+revision=1
+wrksrc="${pkgname}-${version}-src"
+only_for_archs="i686 x86_64"
+hostmakedepends="unzip"
+makedepends="openjdk apache-ant"
+depends="virtual?java-runtime shared-mime-info desktop-file-utils libXext libXrender"
+short_desc="Is a free architectural design application"
```
Comment (Cogitri) [inline srcpkgs/SweetHome3D/template]: `Free architectural design application` sounds better

File: srcpkgs/SweetHome3D/template [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+homepage="www.sweethome3d.com"
+distfiles="${SOURCEFORGE_SITE}/sweethome3d/${pkgname}-${version}-src.zip"
+checksum=43645e34c94bf5b514974428c4a0eefae403fcaf52258cbb9c6fe07d4da8f7db
+
+do_build() {
+	ant application textures furniture examples help java3dLibraries
+}
+
+do_install() {
+	vinstall "${FILESDIR}/${pkgname}.desktop" 755 usr/share/applications
```
Comment (Chocimier) [inline srcpkgs/SweetHome3D/template]: Value of `${pkgname}` never changes, please use it directly.

Comment (Chocimier) [reply srcpkgs/SweetHome3D/template]: Please also expand all other occurrences of `${pkgname}` in this and future templates.

Comment (Chocimier) [reply srcpkgs/SweetHome3D/template]: In wrksrc, distfiles and icon name too.

**Comment by Chocimier** [comment]
Where do the launching script come from? It would be better to grab it with distfiles if from upstream.

File: srcpkgs/SweetHome3D/template [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Template file for 'SweetHome3D'
+pkgname=SweetHome3D
+version=6.1.2
+revision=1
+wrksrc="SweetHome3D-${version}-src"
+only_for_archs="i686 x86_64"
```
Comment (Johnnynator) [inline srcpkgs/SweetHome3D/template]:
```suggestion
archs="i686 x86_64"
```

**Pull request merged**